sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!> @file
+!! @brief Process vertical profile climatologic data for WAM.
+!!
+!! This file contains all data need to form exosphere and used
+!! whole atmsopheric modeling (WAM). it has two modules and some
+!! routines to compute the temperature and compositions of neutral
+!! density in specific values.
+!! the original package contains fortran 77 blockdata and common statement
+!! they are all recoded to use modules and use-statements.
+!!
+!! @author Hann-Ming Henry Juang  NCEP/EMC
+
+!-----------------------------------------------------------------------
+!> Use moduke for blockdata gtd7bk
+!! @author Hann-Ming Henry Juang
+   module wam_gtd7bk_mod
+!          msise-00 01-feb-02                                           
+!x----common/parm7/
+      real ::      pt1(50),pt2(50),pt3(50),pa1(50),pa2(50),pa3(50),     &
+       pb1(50),pb2(50),pb3(50),pc1(50),pc2(50),pc3(50),                 &
+       pd1(50),pd2(50),pd3(50),pe1(50),pe2(50),pe3(50),                 &
+       pf1(50),pf2(50),pf3(50),pg1(50),pg2(50),pg3(50),                 &
+       ph1(50),ph2(50),ph3(50),pi1(50),pi2(50),pi3(50),                 &
+       pj1(50),pj2(50),pj3(50),pk1(50),pl1(50),pl2(50),                 &
+       pm1(50),pm2(50),pn1(50),pn2(50),po1(50),po2(50),                 &
+       pp1(50),pp2(50),pq1(50),pq2(50),pr1(50),pr2(50),                 &
+       ps1(50),ps2(50),pu1(50),pu2(50),pv1(50),pv2(50),                 &
+       pw1(50),pw2(50),px1(50),px2(50),py1(50),py2(50),                 &
+       pz1(50),pz2(50),paa1(50),paa2(50)                                
+!x----common/lower7/
+      real :: ptm(10),pdm(10,8) 
+!x----common/mavg7/
+      real :: pavgm(10) 
+!x----common/datim7/
+      character*4:: isdate(3),istime(2),name(2) 
+!x----common/metsel/
+      integer ::    imr 
+!x----common/pres/
+      real ::     pr65(2,65),pr151(2,151) 
+
+      data imr/0/ 
+      data isdate/'01-f','eb-0','2   '/,istime/'15:4','9:27'/ 
+      data name/'msis','e-00'/ 
+!         temperature                                                   
+      data pt1/                                                         &
+        9.86573e-01, 1.62228e-02, 1.55270e-02,-1.04323e-01,-3.75801e-03,&
```

Comment:
Can you document (perhaps in the module docblock) where these numbers came from?

Reply:
ok. I can try.